Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -149,11 +149,6 @@
 @property (nonatomic, strong) BUYShippingRate *shippingRate;
 
 /**
- *  Shipping rate identifier
- */
-@property (nonatomic, readonly) NSString *shippingRateId;
```

Comment:
We can mark this property as deprecated, and implement a getter which returns shippingRate.shippingRateIdentifier to maintain backward compatibility